.eslintrc.json

```diff
@@ -17,18 +17,30 @@
     "prefer-const": ["error", {
       "destructuring": "all"
     }],
+    "standard/no-callback-literal": "off",
     "node/no-deprecated-api": 0
   },
   "parserOptions": {
     "ecmaVersion": 6,
     "sourceType": "module"
   },
+  "globals": {
+    "isCI": "readonly",
```

Comment:
did we remove this?

Reply:
yep, this PR came before that got merged. will fix